src/views/tables/FillsTable.tsx

```diff
@@ -348,85 +348,87 @@ type StyleProps = {
   withInnerBorders?: boolean;
 };
 
-export const FillsTable = ({
-  columnKeys,
-  columnWidths,
-  currentMarket,
-  initialPageSize,
-  withGradientCardRows,
-  withOuterBorder,
-  withInnerBorders = true,
-}: ElementProps & StyleProps) => {
-  const stringGetter = useStringGetter();
-  const dispatch = useAppDispatch();
-  const { isMobile } = useBreakpoints();
+export const FillsTable = forwardRef(
```

Comment:
wheres the ref though! you should pass through to $Table right?

Reply:
The parent is passing ref even when it didn't receive one, so we just need a component that is the right runtime type to accept it. We're not actually using it. I guess we can handle it so that if we ever try to use it in the future it works.

Reply:
hm thats kind of whack, just remove the parent passing in the ref prop? i think its weird to redefine the child to take in a ref when it doesnt actually pass through

Reply:
we can't because this react-aria tab thing passes ref no matter what